crypto/default/src/main/java/org/keycloak/crypto/def/DefaultCryptoProvider.java

```diff
@@ -63,6 +64,9 @@ public DefaultCryptoProvider() {
         providers.put(CryptoConstants.ECDH_ES_A128KW, new BCEcdhEsAlgorithmProvider());
         providers.put(CryptoConstants.ECDH_ES_A192KW, new BCEcdhEsAlgorithmProvider());
         providers.put(CryptoConstants.ECDH_ES_A256KW, new BCEcdhEsAlgorithmProvider());
+        providers.put(CryptoConstants.MLDSA44, new BouncyCastlePQCProvider());
+        providers.put(CryptoConstants.MLDSA65, new BouncyCastlePQCProvider());
+        providers.put(CryptoConstants.MLDSA87, new BouncyCastlePQCProvider());
```

Comment:
Remove, this is not needed if you change to `ML-DSA-**` as those are available in BouncyCastleProvider